### ISSUE TITLE
There is a problem with the number of samples in the json file

### ISSUE THREAD

**Issue by GG-Bond-007** [clovaai/rainbow-memory#4]
On the CIFAR10 data set, there are a total of 5 tasks, and each task has two classes, so the number of samples in the training set for each task should be 10,000, but why is the number of samples in the json file 9750?

**Comment by hwany-j** [comment]
First, thank you for visiting our repository, and sorry for the late reply. Since we used the validation set on CIFAR-10 like [experiments of Gdumb paper](https://www.robots.ox.ac.uk/~tvg/publications/2020/gdumb.pdf), the number of samples in the json file is 9750 for each as you mentioned. For easily using our json files, I added the validation set of which the number of samples is 250, and you can combine the dataset if you want to make 10,000. 
I hope this reply and updated code will help your research and development 🙏
